Tools/Scripts/libraries/resultsdbpy/resultsdbpy/bug_trackers/bugzilla.py

```diff
@@ -0,0 +1,117 @@
+from urllib.parse import quote
+from resultsdbpy.bug_trackers.bug_description import translate_selected_dots_to_bug_title_and_description
+from resultsdbpy.controller.bug_tracker_controller import BugTrackerConfig
+
+class Bugzilla(BugTrackerConfig):
+
+    def __init__(self):
+        super().__init__('bugzilla')
+
+    def create_bug(self, content):
+        """
+            content formats: 
+            {
+                "selectedRows": [{
+                    "config": {
+                        "model":  ...,
+                        "architecture": ...,
+                        "style": ...,
+                        "flavor": ...,
+                        "sdk": ...,
+                        "version_name": ...,
+                        "suite": ...
+                    },
+                    "results": [
+                        {
+                            "uuid": ...,
+                            "actual": ...,
+                            "expected": ...,
+                            "start_time": ...,
+                            "time": ...
+                        }
+                    ]
+                }],
+                "willFilterExpected": ...,
+                "repositories": [],
+                "suite": ...,
+                "test": ...
+            }
+            
+        """
+
+        selected_rows = content['selectedRows']
+        will_filter_expected = content['willFilterExpected']
+        repositories = content['repositories']
+        suite = content['suite']
+        test = content['test']
+
+        title_components, description_components = translate_selected_dots_to_bug_title_and_description(self.commit_context, selected_rows, test, suite, repositories, will_filter_expected)
+
+        component = 'New Bugs'
+        version = 'WebKit Nightly Build'
+        if suite == 'api-tests':
+            component = 'WebKit API'
+        if suite == 'webkitpy-tests':
+            component = 'Tools / Tests'
+        if suite == 'internal-media-tests':
+            component =  'Media'
+        if suite == 'javascriptcore-tests':
+            component = 'JavaScriptCore'
+        if test and any([suite == 'layout-tests', suite == 'internal-security-tests']):
```

Comment:
A thought I had with this series of `if` statements: most of these basically entail looking through our bugs.webkit.org components and checking test names for the component names. I wonder if leveraging `bugzilla.Tracker('https://bugs.webkit.org').projects` could improve this git of code, especially if you split the test name by `/` and then just checked to see if the component name was anywhere in the path. Would have the added bonus of being resilient to future component additions, and even compatible with a future GitHub workflow, since we have a similar `.project` property on that object too.